src/sentry/static/sentry/app/views/settings/organization/auth/providerItem.jsx

```diff
@@ -16,23 +19,43 @@ export default class ProviderItem extends React.PureComponent {
   };
 
   handleConfigure = e => {
-    this.props.onConfigure(this.props.providerKey, e);
+    if (typeof this.props.onConfigure === 'function') {
```

Comment:
why was this needed?

Reply:
1) Being defensive
2) Change of API (at the time this component was added, we had planned to change the configuration form into react, but now we are leaving it in django)